armagetronad-appdata.patch

```diff
@@ -1,33 +1,46 @@
 diff --git a/desktop/armagetronad.appdata.xml.in b/desktop/armagetronad.appdata.xml.in
-index fb8e34ff..dcbc0c49 100644
+index 516025448..08be527af 100644
 --- a/desktop/armagetronad.appdata.xml.in
 +++ b/desktop/armagetronad.appdata.xml.in
-@@ -24,16 +24,24 @@
+@@ -4,7 +4,16 @@
+   
+   <name>@progtitle@</name>
+   <summary>3D Lightcycle Game</summary>
+-  
++  <developer id="tld.vendor">
```

Comment:
@zmanuel The id should be an actual id, not this example one.

Reply:
`org.armagetronad` should be the correct id.

Reply:
Ah, thanks for the correction. Sorry, as you can tell, I try to interact with these metadata systems as little as possible.